src/openrct2-ui/windows/TopToolbar.cpp

```diff
@@ -1168,7 +1168,9 @@ static void sub_6E1F34(
         return;
     }
 
+    // The upper byte of selected_scenery contains the type, the lower byte the index.
     uint8_t scenery_type = selected_scenery >> 8;
+    selected_scenery &= 0xFF;
```

Comment:
This could be done in the line above

Reply:
Uhm, how? The line above assigns the `scenery_type` variable, which is something else.

Reply:
Nevermind, been looking at the wrong variable.